magento/magento2
# magento#38805: Issue at product import when provided with custom options-type: file

Ensure the import with custom options works properly for the 'file' type.





### Description (*)
- Fixed the issue with parsing 'file_extension' data.
- Resolved the issue where the price was not set for the custom-options file type.

### Fixed Issues (if relevant)

1. Fixes magento/magento2#38805

### Manual testing scenarios (*)


1. Create a CSV file for product import.
2. Add the necessary data for the product along with custom options.
3. Add the type file and other data for the custom options column. Ensure you include more than one file_extension attribute.
4. Navigate to System > Import.
5. Choose Entity type: Products and Import behavior: Add/Update. Leave the remaining settings as default (separators: ",").
6. Validate and import the product.
7. Check the created product customization options section on the edit product page. Ensure the price is set and file extensions are displayed properly.

**Attached the CSV used during the testing.**
[catalog_product.csv](https://github.com/user-attachments/files/16192938/catalog_product.csv)


### Questions or comments


### Contribution checklist (*)
